<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BTC Prediction LSTM</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="tickerLabel">
        <property name="text">
         <string>Ticker:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="tickerInput">
        <property name="text">
         <string>BTC-USD</string>
        </property>
        <property name="placeholderText">
         <string>Enter Ticker</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="dateLayout">
      <item>
       <widget class="QLabel" name="startDateLabel">
        <property name="text">
         <string>Start Date:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="startDateInput">
        <property name="placeholderText">
         <string>YYYY-MM-DD</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="endDateLabel">
        <property name="text">
         <string>End Date:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="endDateInput">
        <property name="placeholderText">
         <string>YYYY-MM-DD</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="dataPreparationButton">
        <property name="text">
         <string>DATA PREPARATION</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="featureEngineeringButton">
        <property name="text">
         <string>FEATURE ENGINEERING</string>
        </property>
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="trainButton">
        <property name="text">
         <string>TRAIN</string>
        </property>
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton">
        <property name="text">
         <string>STOP TRAINING</string>
        </property>
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="predictButton">
        <property name="text">
         <string>PREDICT</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="logOutput">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
